config/initializers/git_info.rb

```diff
@@ -0,0 +1,7 @@
+module Git
+    `which git`
+    if $?.success?
+      BRANCH = `git symbolic-ref HEAD 2>/dev/null | cut -d"/" -f 3`.strip
+      COMMIT = `git log --pretty=format:'%h' -n 1`.strip
+    end
```

Comment:
what about displaying also the version in 

https://github.com/SUSE/Portus/blob/master/VERSION